### ISSUE TITLE
Aria-live attribute inappropriate for player controls

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/sul-embed#2237]
### Current behavior

The `aria-live="polite"` attribute on player controls causes the screen reader to announce button names repeatedly upon selection. For example, when clicking the "Play" button, the screen reader announces "Play Play Play." Similarly with "Pause Pause Pause," "Mute Mute Mute," so on).

### Steps to reproduce

Use a screen reader to interact with an embedded [video item](https://purl.stanford.edu/qf378nj5000) or [audio item](https://purl.stanford.edu/bb158br2509).

**Note**: The described behavior appears in the Chrome and Firefox browsers, but not in Safari.

### Possible solution

Removing the `aria-live="polite"` attribute results in the screen reader only announcing the button name a single time, as expected.

### Reported by SODA

> Violation: Avoid inappropriate use of ARIA roles, states, and properties
> Severity: Minor
> 
> [Issue]
> There is an aria-live region on a button which is most likely inappropriate. 
> 
> [User Impact]
> When inappropriate ARIA roles, states, and/or properties are used it may prevent users of assistive technology from correctly interpreting and interacting with the content.
> 
> [Code Reference]
> ```
> <button class="vjs-fullscreen-control vjs-control vjs-button" type="button" title="Fullscreen" aria-disabled="false">
>         <span class="vjs-icon-placeholder" aria-hidden="true"></span>
>         <span class="vjs-control-text" aria-live="polite">Fullscreen</span>
> </button>
> 
> ```
> [Recommendation]
> Developers must avoid inappropriate use of ARIA roles, states, and/or properties. The aria-live region is most likely inappropriate for this element, but further testing may be needed.

**Comment by dnoneill** [comment]
I looked around and it does look like this is set in video.js. https://videojs.com/blog/feature-spotlight-accessibility/

**Comment by taylor-steve** [comment]
@alundgard et al, I added more context to the remote issue: https://github.com/videojs/video.js/issues/8912. Tagging you just in case you've got any thoughts on approach here. It's a tricky issue.

I think removing `aria-live` as SODA suggests isn't a possible solution for the reason I mentioned in the issue. Technically if we are only worried about controlling the video via direct interaction with the controls, we could maintain a custom version of Video.js or try and hook that behavior to alter it. That'd be its own kind of nightmare though, people might throw virtual tomatoes at me.

**Comment by jcoyne** [comment]
🍅

**Comment by dnoneill** [comment]
https://github.com/videojs/video.js/issues/8912

**Comment by corylown** [comment]
I met with SODA about this today. They have a contact with one of the original developers of video.js and will reach out to other video accessibility experts and get back to me with more information and recommendations about how to fix this.

**Comment by dnoneill** [comment]
@corylown Not sure if you have heard back from SODA. If you have could you put the status of that in the ticket?

**Comment by corylown** [comment]
I have not heard anything. I'll contact them if I don't hear soon.
